### PR TITLE
Manifest edits

### PR DESCRIPTION
**Pull request by ben-bay**

### CODE REVIEW

**Comment by lucpeterson** [comment]
What's this trying to fix?

**Comment by koning** [comment]
It is fixing spack test

**Comment by ben-bay** [comment]
> It is fixing spack test

Will this be okay?

**Comment by koning** [comment]
Ar you asking if it worked? I can see if I can make the changes and have spack pull from the cache to test. We might need to explicitly include the test files since it looks for files of the form test/test*.py and ours are tests/*. I don't think the prune Makefile is necessary since this is not one of the files python includes by default.

**Comment by ben-bay** [comment]
> Are you asking if it worked? I can see if I can make the changes and have spack pull from the cache to test. We might need to explicitly include the test files since it looks for files of the form test/test*.py and ours are tests/*. I don't think the prune Makefile is necessary since this is not one of the files python includes by default.

Updated.

**Comment by koning** [comment]
Actually I was wrong, everything seems to be included in the sdist if you don't exclude it.  I'm checking to see if our tests/ dir will work with spack test, I believe it is expecting test/.

File: MANIFEST.in [hunk 1]
```diff
@@ -1,6 +1,5 @@
-prune docs
-prune tests
 include workflows/*
 include merlin/data/*
 include requirements.txt
 include requirements/*
+include tests/*
```
Comment (koning) [inline MANIFEST.in]: This is not needed.

Comment (ben-bay) [reply MANIFEST.in]: Removed.

**Comment by lucpeterson** [comment]
So is this working now?

**Comment by koning** [comment]
It creates the correct sdist file, but the tests don't work in spack. I sent an email with the error on 12/6.

**Comment by ben-bay** [comment]
@koning are we good to merge this now?

**Comment by koning** [comment]
yes, this works.

**Pull request merged**